src/background/httpClients.ts

```diff
@@ -130,3 +140,55 @@ export const httpClient = {
     return fetch(url, { credentials: 'omit' });
   },
 };
+
+type NonNullFields<T> = {
+  [P in keyof T]: NonNullable<T[P]>;
+};
+
+const _getTeamsInfoQuery = gql`
+  query getTeamsInfo {
+    locales {
+      code
+      name
+      approvedStrings
+      pretranslatedStrings
+      stringsWithWarnings
+      stringsWithErrors
+      missingStrings
+      unreviewedStrings
+      totalStrings
+    }
+  }
+`;
+
+interface GetTeamsInfoResponse {
+  locales: NonNullFields<Required<NonNullFields<GetTeamsInfoQuery>>['locales']>;
+}
+
+const _getProjectsInfoQuery = gql`
+  query getProjectsInfo {
+    projects {
+      slug
+      name
+    }
+  }
+`;
+
+export interface GetProjectsInfoResponse {
+  projects: NonNullFields<
+    Required<NonNullFields<GetProjectsInfoQuery>>['projects']
+  >;
+}
+
+export function graphqlClient(pontoonBaseUrl: string) {
+  const client = getSdk(
+    new GraphQLClient(pontoonGraphQL(pontoonBaseUrl), {
+      method: 'GET',
```

Comment:
Pontoon does not accept the query via POST, only via GET. With POST it returns CSRF error.